src/elem.js

```diff
@@ -312,7 +295,6 @@ exports.text = function(text) {
 
 exports.render = function(el, node, context) {
   Common.markStart('Elem.render');
-  var waitingHandlers = (context || {}).waitingHandlers || [];
```

Comment:
What you can do here is, if waiting handlers already exists in the context, just call the old extractEventHandlers function. if not, don't call it and directly attach the handler. This way components will still works